appimagebuilder/utils/finder.py

```diff
@@ -160,3 +160,27 @@ def _run_check(self, check_function, path):
             passed = check_function(path)
             self.cache[key] = passed
             return passed
+
+    @staticmethod
+    def list_does_not_contain_file(file_list: [pathlib.Path], file: pathlib.Path):
+        allowed = True
+        for item in file_list:
+            if str(item) == str(file):
```

Comment:
why a string conversion  is required if both variables should be of Path type?

Reply:
Sorry, I haven't coded in Python in a while. Fixed!